brainbox/examples/docs_explore_passive.py

```diff
@@ -12,14 +12,14 @@
 import brainbox.io.one as bbone
 import brainbox.task.passive as passive
 
-eid = 'c7248e09-8c0d-40f2-9eb4-700a8973d8c8'
-probe = 'probe01'
-one = ONE()
+eid = '4ecb5d24-f5cc-402c-be28-9d0f7cb14b3a'
+probe = 'probe00'
+one = ONE(base_url='https://openalyx.internationalbrainlab.org')

```

Comment:
It might be worth adding the silent flag to all these, that way the user will never be prompted to set up open alyx

Reply:
Good idea!